src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -134,6 +134,7 @@ def load_arguments(self, _):
     with self.argument_context('backup restore restore-disks') as c:
         c.argument('storage_account', help='Name or ID of the staging storage account. The VM configuration will be restored to this storage account. See the help for --restore-to-staging-storage-account parameter for more info.')
         c.argument('restore_to_staging_storage_account', arg_type=get_three_state_flag(), help='Use this flag when you want disks to be restored to the staging storage account using the --storage-account parameter. When not specified, disks will be restored to their original storage accounts. Default: false.')
+        c.argument('TargetRg', options_list=['--TargetRg', '-t'], help='Use this to specify the target resource group in which the restored disks will be saved')
```

Comment:
To be consistent, use `--target-resource-group`. Argument name should not be in Pascal Naming style